src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -15,14 +15,7 @@
     "NuGet.ProjectModel": "3.6.0-rc-1954"
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Runtime": {
-          "type": "build"
-        }
-      }
-    },
-    "netstandard1.6": {
+    "netstandard1.3": {
```

Comment:
Maybe you should hold on this change for this file. It probably will pick up a dependency on MSBuild and we don't know if they can support netstandard1.3. They did indicate that supporting net451 should be feasible.